From 1ab0174e225b147360addffeba737df88b9ccfee Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@chromium.org>
Date: Fri, 25 May 2018 11:51:38 -0400
Subject: [PATCH 18/26] init: separate pre-startup logic out

We usually use "pre-startup" for setting up state/temp files,
so do the same here to make the code a bit more readable.

BUG=None
TEST=precq passes

Change-Id: Ic33f0c9be39c8f7539d39d0740b91654a1fb9ea2
Reviewed-on: https://chromium-review.googlesource.com/1074531
Commit-Ready: Mike Frysinger <vapier@chromium.org>
Tested-by: Mike Frysinger <vapier@chromium.org>
Reviewed-by: Mike Frysinger <vapier@chromium.org>
---
 init/tlsdated-cros.conf | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/init/tlsdated-cros.conf b/init/tlsdated-cros.conf
index f96ba30..763f1cb 100644
--- a/init/tlsdated-cros.conf
+++ b/init/tlsdated-cros.conf
@@ -13,13 +13,14 @@ respawn
 # have specifically for talking to Google systems.
 env GOOGLE_CERTS=/usr/share/chromeos-ca-certificates
 
-script
+pre-start script
   mkdir -m 755 -p /var/cache/tlsdated
-  # Make sure the timestamp file has the proper permissions.
+  # Make sure the timestamp file has the proper permissions if it exists.
   chmod 0644 /var/cache/tlsdated/timestamp || true
-  # When it runs tlsdate, tlsdated stitches together an argument vector for it
-  # as follows: it begins with everything supplied to it after the --, then
-  # appends -H $host -p $port, and maybe -x $proxy if it has a proxy to use.
-  exec tlsdated -v -- /usr/bin/tlsdate -v -C "$GOOGLE_CERTS" -l \
-      2>&1 | logger -t tlsdate
 end script
+
+# When it runs tlsdate, tlsdated stitches together an argument vector for it
+# as follows: it begins with everything supplied to it after the --, then
+# appends -H $host -p $port, and maybe -x $proxy if it has a proxy to use.
+exec tlsdated -v -- /usr/bin/tlsdate -v -C "$GOOGLE_CERTS" -l \
+    2>&1 | logger -t tlsdate
-- 
2.20.1

